src/scenes/App/components/TermsAndConditions/index.jsx

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
```

Comment:
This should be a `.js` not a `.jsx` to keep things consistent

Reply:
thought all react components should be `.jsx`? fixed